plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -2944,7 +2949,7 @@ function woocommerce_form_field( $key, $args, $value = null ) {
 						$field .= '<option value="' . esc_attr( $ckey ) . '" ' . selected( $value, $ckey, false ) . '>' . esc_html( $cvalue ) . '</option>';
 					}
 
-					$field .= '</select>';
+					$field .= esc_attr( $asterisk ) . '</select>';
```

Comment:
Since `$asterisk` isn't an attribute but is full HTML, `esc_attr` isn't the proper escaping function here. We may be able to leave this since we're not actually echoing anything here, just storing that in the `$field` variable where I assume it get's echoed later